### PR TITLE
Migrate order id from int to UUID

### PR DESCRIPTION
**Pull request by IKarbowiak**
Change order `id` from `int` to `UUID`.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-5849 Migrate order IDs to UUID](https://app.clickup.com/t/2549495/SALEOR-5849)

File: saleor/graphql/order/schema.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ class OrderQueries(graphene.ObjectType):
     )
     order_by_token = graphene.Field(
         Order,
-        description="Look up an order by token.",
+        description="Look up an order by token. {DEPRECATED_IN_3X_FIELD}",
```
Comment (korycins) [inline saleor/graphql/order/schema.py:111]: IMHO `DEPRECATED_IN_3X_FIELD` should be placed at the beggining of description. (It is done in that way for the rest of depreceated descriptions)

Comment (IKarbowiak) [reply saleor/graphql/order/schema.py:111]: Will be changed in #9295

**Review by L3str4nge — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Comment by jiapei100** [comment]
```console
django.db.utils.ProgrammingError: relation "order_order_number_seq" does not exist
```

```console
def get_order_number():
    with connection.cursor() as cursor:
        cursor.execute("SELECT nextval('order_order_number_seq')")
        result = cursor.fetchone()
        return result[0]
```
this part of file `graphql/order/models.py' wouldn't work... Any suggestions?

**Pull request merged**